lib/server_env.rb

```diff
@@ -4,4 +4,8 @@ module ServerEnv
   def self.instance_index
     CloudFoundry.instance_index || 0
   end
+
+  def self.first_instance?
+    self.instance_index == 0
```

Comment:
looks like we are doing this check elsewhere (`unless ServerEnv.instance_index == 0`) -- update that call to use `first_instance?`